very_good_core/hooks/pubspec.yaml

```diff
@@ -8,6 +8,7 @@ dependencies:
   mason: ^0.1.0-dev.52
 
 dev_dependencies:
+  clock: ^1.1.1
```

Comment:
I think we can now make it a main dependency (instead of dev), since it is being used in the source code, not just tests.

Reply:
I've made the change and moved the clock package from dev_dependencies to dependencies. Thanks for the suggestion!